packages/components/src/Typeahead/Typeahead.scss

```diff
@@ -78,20 +78,30 @@ $tc-typeahead-items-font-size: 1.5rem !default;
 		display: flex;
 		align-items: center;
 
+		.typeahead-input {
+			padding-right: 2.5em;
```

Comment:
For both this and the other padding-right: I think we normally use "rem" instead of "em", but I saw "em" already elsewhere so maybe it causes issues?

Reply:
Yes it has not the same effect, so I prefer to keep `em` for now thx